resources/property-names.edn

```diff
@@ -229,6 +229,8 @@
  [:cg/clinvar-trait-set-id "http://dataexchange.clinicalgenome.org/terms/clinvar_trait_set_id"]
  [:cg/medgen-id "http://dataexchange.clinicalgenome.org/terms/medgen_id"]
  [:cg/variation-id "http://dataexchange.clinicalgenome.org/terms/variation_id"]
+ [:cg/deleted "http://dataexchange.clinicalgenome.org/terms/deleted"]
+ [:cg/total_copies "http://dataexchange.clinicalgenome.org/terms/total_copies"] ; CNV normalization param
```

Comment:
This is a little incongruent with the rest of the terms... I believe most use "-" vs. "_"